docker-compose.yml

```diff
@@ -2,12 +2,16 @@ version: "3.3"
 services:
   lobby:
     restart: always
-    build:
-      context: .
-    image: keyteki-lobby
+    build: .
+    image: keyteki
     ports:
       - "4000:4000"
       - "9229:9229"
+    volumes:
```

Comment:
Just doing:

```
volumes:
  - ".:/usr/src/app"
```

Didn't seem to work, I didn't dig through it too much but in retrospect it may have had something to do with trying to use my local node modules within the container.